docs/source/noaa_csp.rst

```diff
@@ -40,160 +51,252 @@ Login to the NOAA CSP
+2. In the `Mount Point` box, name it `/lustre` (the common and default choice) as pointed by red-arrow 2.
+   If you choose a different name, make sure to make the Global-Workflow setup step
+   use the name chosen here.
+
+If you have a `S3 bucket`, one can attached as:
+
+3. In the `Storage` box, select the bucket you want to use, as in red-arrow 3.
+4. In the `Mount Point` box, name it `/bucket` (the common and default choice) as pointed by red-arrow 4.
+
+.. image:: _static/noaacsp_filesystem_4.png
+
+Always remember to click `Save Changes` after making any changes to the cluster.
 
 **************************
 Using the NOAA CSP Cluster
 **************************
 
-To activate the cluster, click the button circled in
-:red-text:red. The cluster status is denoted by the color-coded button
-on the right. The amount of time required to start the cluster is
-variable and not immediate and may take several minutes for the
-cluster to become.
+To activate the cluster, click the `cluster` at right panel of the noaa.parallel.works web-site,
+as point by the red-arrow. Then click the `Sessions` button in red-square, and than click the
+button in red-circle. The cluster status is denoted by the color-coded button
+on the right: red, stopped; orange, requested; green, active. The amount of time required to start
+the cluster varies and not immediate and may take several minutes for the cluster to become active.
 
 .. image:: _static/noaacsp_using_1.png
 
-For instances where a NOAA CSP cluster does not initialize, useful
-output can be found beneath the ``Logs`` section beneath the
-``Provision`` tab as illustrated below. Once again, when opening
-issues related to the NOAA CSP cluster initialization please include
-this information.
+when the cluster is activate, user will see:
+1. Green dot means the cluster is active, pointed by red-arrow 1.
+2. Green dot means the cluster is active, pointed by red-arrow 2.
+3. Green button means the cluster is active, pointed by red-arrow 3.
+4. Click the blue-square with arrow inside pointed by red-arrow 4 will copy the cluster's IP into clipboard,
+   which you can open a laptop xterm/window, and do `ssh username@the-ip-address` in the xterm window will connect you
+   to the AWS cluster, and you can do you work there.
+5. Which is the `username@the-ip-address`, or your AWS PW cluster. Click it, will have a PW web terminal appear in the
+   bottom of the web-site, which you can work on this terminal to use your AWS cluster.
+
+Please note, as soon as the cluster is activated, AWS/PW starts charging you for use the cluster.
+As this cluster is exclusive for yourself, AWS keep charging you as long as the cluster is active.
+For running global-workflow, one need to keep the cluster active if there is any rocoto jobs running,
+as rocoto is using `crontab`, which needs the cluster active all the time, or the crontab job will be terminated.
```

Comment:
```suggestion
Please note, as soon as the cluster is activated, AWS/PW starts charging you for use of the cluster.
As this cluster is exclusive for yourself, AWS will keep charging you as long as the cluster is active.
For running global-workflow, one needs to keep the cluster active if there are any Rocoto jobs running,
as Rocoto is using `crontab`, which needs the cluster active all the times, or the crontab job(s) will be terminated.
```

Reply:
@weihuang-jedi Has this comment been addressed yet?